methods/src/lib.rs

```diff
@@ -14,3 +14,40 @@
 
 //! Generated crate containing the image ID and ELF binary of the build guest.
 include!(concat!(env!("OUT_DIR"), "/methods.rs"));
+
+#[cfg(test)]
```

Comment:
I'm not sure this is a good place for unit tests

Reply:
I do agree, but have not thought of a better place. Maybe put them in a `test` directory? If you have any ideas, that would be helpful.